components/Archetype/interfaces.rst

```diff
@@ -0,0 +1,20 @@
+Interfaces
+==========
+
+.. note::
+
+    For more detailed information about the interfaces go to `Sylius Api`_.
+
+.. _component_archetype_model_archetype-interface:
+ArchetypeSubjectInterface
+-------------------------
+
+To characterize an object with attributes and options from a common archetype, the object class needs to implement
+the ``ArchetypeSubjectInterface``.
+
+.. note::
+
+    You will find more information about that interface in `Sylius Api ArchetypeSubjectInterface`_.
+
+.. _Sylius Api: http://api.sylius.org/Sylius/Component/Archetype/Model.html
```

Comment:
This line should be placed under https://github.com/Sylius/Sylius-Docs/pull/329/files#diff-61a28ac3c5986aede819edaafe1a3867R6